migrate/live-migration/live-migration-from-postgres.md

```diff
@@ -120,19 +118,14 @@ it will start `ANALYZE` on the target database. This updates statistics in the
 target database, which is necessary for optimal querying performance in the
 target database. Wait for `ANALYZE` to complete.
 
-<Highlight type="important">
-Application downtime begins here.
-</Highlight>
+## 4. Validate the data in target database and use it as new primary
 
-Once the lag between the databases is below 30 megabytes, and you're ready to
-take your applications offline, stop all applications which are writing to the
-source database. This is the downtime phase and will last until you have
-completed the validation step (4). Be sure to go through the validation step
-before you enter the downtime phase to keep the overall downtime minimal.
+Once the lag between the databases is below 130 megabytes, we recommend performing data integrity checks. There are two ways to do this:
 
-Stopping writes to the source database allows the live migration process to
-finish replicating data to the target database. This will be evident when the
-replication lag reduces to 0 megabytes.
+1. With downtime: Stop database operations from your application, which will result in downtime. This allows the live migration to catch up on the lag between the source and target databases, enabling the validation checks to be performed. The downtime will last until the lag is eliminated and the data integrity checks are completed.
+2. Without downtime: Since the difference between the source and target databases is less than 130 MB, you can perform data integrity checks, excluding the latest data that is still being written. This approach does not require taking your application down.
```

Comment:
I don't think we should make this change. We should let the users have 100% confidence on data integrity and not promote "partial integrity" by comparing non-recent data, otherwise he may have doubts later on regarding integrity. Note, the users will always have to take momentary downtime to switch applications writing from source to target.